wrappers/Julia/CoolProp.jl

```diff
@@ -55,6 +55,7 @@ end
 
 # CoolProp::get_parameter_information_string
 function get_parameter_information_string(Key::AbstractString)
+message_buffer[1:4] = UInt8[108 111 110 103] # "long" written in UInt8
```

Comment:
@mikekaganski You are right, the error caching is a bit obsolete now, as there is two possible wrong parameters.
```C++
catch(...){
str2buf(format("parameter is invalid: %s", param), Output, n);
}
```
I think it would be good that I also update that ^^